Tests/SonarScanner.MSBuild.PreProcessor.Tests/SonarWebServiceTest.cs

```diff
@@ -99,6 +99,35 @@ public void LogWSOnError()
             }
         }
 
+        [DataTestMethod]
+        [DataRow("7.9.0.5545", DisplayName ="7.9 LTS")]
+        [DataRow("8.0.0.18670", DisplayName = "SonarCloud" )]
+        [DataRow("8.8.0.1121")]
```

Comment:
Let's add 9.0 for completeness and 10.0 to be sure that 10>7. As it comes from string, funny stuff (should not but) could happen.